README.md

```diff
@@ -134,6 +134,7 @@ Let's create some random data and index it:
 
 ```python
 from jina import Document
+import numpy
```

Comment:
please move numpy import before jina to respect standard import order

Reply:
good spot, reordered